recipes/fast-dp/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "fast-dp" %}
```

Comment:
You don't need to change this now, but for future reference, you can actually use `{% set name = "fast_dp" %}` here so you can use that in the `url` section. Then, in the `package` section you can have `name: {{ name|replace("-", "_") }}`.

Reply:
I'll do it right away as I tend to copy and paste from existing recipes :)